README.md

```diff
@@ -161,3 +161,12 @@ uvicorn therapy.main:app --reload
 Next, view the OpenAPI docs on your local machine:
 
 http://127.0.0.1:8000/therapy
+
+
+### FAQ
+
+**A data import method raised a SourceFormatException instance. How do I proceed?**
+
+TheraPy will automatically try to acquire the latest version of data for each source, but sometimes, sources alter the structure of their data (e.g. adding or removing CSV columns). If you encounter a SourceFormatException while importing data, please notify us by creating a new [issue](https://github.com/cancervariants/therapy-normalization/issues) if one doesn't already exist, and we will attempt to resolve it.
+
+In the meantime, you can force TheraPy to use an older data release by removing the incompatible version from the source data folder, manually downloading and replacing it with an older version of the data per the structure described above, and calling the CLI with the `--use_existing` argument.
```

Comment:
I'm going to reorganize the entire README at some point -- there's an issue for it somewhere -- but I'm stashing this at the bottom for now. Do you think this description makes sense?